src/lib/info-cache.js

```diff
@@ -17,8 +15,10 @@ class InfoCache {
       // Default to Five Bells Ledger
       // TODO: This hack is necessary to fetch the precision information for
       // non-adjacent ledgers. How can we handle this case properly?
+      const FiveBellsLedger = require('ilp-plugin-bells')
```

Comment:
That change is from 'remove ledger specific code,' so it should probably be addressed in pull request 168. I'm going to rebase my commit on top of that once it's merged.